_posts/2022-06-14-infinispan-14-indexing-query-news.adoc

```diff
@@ -70,24 +70,23 @@ public class Author {
 }
 ----
 
-Indexed fields without any special string/text transformation will be annotated as `@Basic`.
-If we need to apply a normalizer to a String field, we will opt for a `@Keyword` annotation.
-If we need to apply an analyzer to a String field, we will opt for a `@Text` annotation.
+Use the `@Basic` annotation for indexed fields without any special string/text transformation.
+When you apply a normalizer to a String field, use the `@Keyword` annotation.
+When you apply an analyzer to a String field, use the `@Text` annotation.
 
-The new annotations allow setting with the same annotation if the field should be sortable, or projectable, or its normalizer or its analyzer.
-However, not all the combinations will be possible, for instance the attribute sortable is not present on the `@Text` annotation,
-since an analyzed field cannot be used to sort the result set.
+You can use the same annotations for the fields that are sortable or projectable and for its normalizer or analyzer.
```

Comment:
Thank you for the explanation. What do you think about:
The new annotations allows you to set a single annotation per indexed field if the field should be `sortable` or `projectable` and if you want to use an `analyzer' or 'normalizer' on the field.
However, the combination of an attribute and the annotation must be supported, for instance the attribute `sortable` cannot be used with `@Text` annotation, since an analyzed field cannot be used to sort the result set.